docs/dev/release.md

```diff
@@ -154,6 +154,43 @@ This follows the same general release procedure as `main` with a few extra steps
 
 See procedure for `pass-core`, but without Docker images
 
+## JavaScript projects
+
+The following projects can be released by performing the following steps.
+
+### `pass-ui`
+
+Update the version in `package.json` and in `build.sh`.
```

Comment:
Not sure if we want to be explicit about needing to commit the version bump and push to GH